docker/Dockerfile-tf2

```diff
@@ -3,30 +3,29 @@ FROM twosixarmory/base:${base_image_tag} as armory-base
 ARG armory_version
 ARG armory_build_type
 
-RUN /opt/conda/bin/pip install --no-cache-dir \
-    adversarial-robustness-toolbox==1.9.1
-
 # This section installs tensorflow object detection API from
 # https://github.com/tensorflow/models/blob/79354e14a4b41ff9019f4a5ebd12cfa498917182/research/object_detection/packages/tf2/setup.py
 # It is messy
-RUN /opt/conda/bin/conda install protobuf==3.14.0 && \
-    /opt/conda/bin/conda clean --all
+#RUN /opt/conda/bin/conda install protobuf==3.14.0 && \
```

Comment:
the general recommendation is to let git keep track of dead code, not the code itself

Reply:
This dead code I actually meant to remove, as it is now installed in base.